### DISCUSSION TITLE
谁知道安卓客户端的日志在哪里找

### DISCUSSION THREAD

**Discussion by woshichenghaibo** [2dust/v2rayNG#4185] [Q&A]
我想提取出来反馈问题用

**Comment by 2dust** [comment]
左边有个logcat

**Reply by swuecho** [reply]
@woshichenghaibo 这个方法最简单， 侧边栏可以看到。 我当时忽略了。 安adb太麻烦了

**Comment by swuecho** [comment]
1. 安装 adb https://developer.android.com/tools/releases/platform-tools
2. https://jingyan.baidu.com/article/656db9184018b5a380249c3e.html
3. ./adb logcat -s com.v2ray.ang GoLog V2rayConfigUtilGoLog Main

**Comment by woshichenghaibo** [comment]
收到，谢谢！
________________________________
发件人: Hao Wu ***@***.***>
发送时间: 2025年1月6日 20:26
收件人: 2dust/v2rayNG ***@***.***>
抄送: woshichenghaibo ***@***.***>; Author ***@***.***>
主题: Re: [2dust/v2rayNG] 谁知道安卓客户端的日志在哪里找 (Discussion #4185)


  1.  安装 adb https://developer.android.com/tools/releases/platform-tools
  2.  https://jingyan.baidu.com/article/656db9184018b5a380249c3e.html
  3.  ./adb logcat -s com.v2ray.ang GoLog V2rayConfigUtilGoLog Main

―

**Comment by woshichenghaibo** [comment]
不一样。侧边栏那里没几行
________________________________
发件人: Hao Wu ***@***.***>
发送时间: 2025年1月16日 15:42
收件人: 2dust/v2rayNG ***@***.***>
抄送: woshichenghaibo ***@***.***>; Mention ***@***.***>
主题: Re: [2dust/v2rayNG] 谁知道安卓客户端的日志在哪里找 (Discussion #4185)


@woshichenghaibo<https://github.com/woshichenghaibo> 这个方法最简单， 侧边栏可以看到。 我当时忽略了。 安adb太麻烦了

―

**Reply by swuecho** [reply]
我还以为一样的，那就adb 吧
